tensorflow/core/distributed_runtime/rpc/grpc_channel.cc

```diff
@@ -269,7 +269,7 @@ class SparseGrpcChannelCache : public CachingGrpcChannelCache {
         job_id_(job_id),
         host_ports_(host_ports),
         channel_func_(std::move(channel_func)) {
-    LOG(INFO) << "Initialize GrpcChannelCache for job " << ToString();
+    VLOG(2) << "Initialize GrpcChannelCache for job " << ToString();
```

Comment:
Why not use `LOG(DEBUG)` instead to keep using the same factory of functions?

(driving by review as I notice this PR has issues importing into google codebase)

Reply:
(ignore the issues part, seems to be resolved now)

Reply:
Seems like all other logs are VLOG in this file. Would it be appropriate to use VLOG or LOG(DEBUG) from your point of view?

Reply:
Oh, in that case for consistency, please use `VLOG`. I didn't realize that.